stylesheets/_component.progress-bars.scss

```diff
@@ -39,3 +39,77 @@
 .progress-bar--compact {
   height: 5px;
 }
+
+.stepped-progressbar {
+  background-color: color(gray, lightest);
+  display: table;
+  padding: 20px;
+  width: 100%;
+
+  .step {
+    display: block;
+    float: left;
+    overflow: hidden;
+    position: relative;
+    width: 50px;
+
+    span {
```

Comment:
`step-box` rather than `span`?

Reply:
There is a complex structure to create the desired result with the line in the back back not just a background. Span gives enough control

Reply:
My reasoning behind it was what if other `span`'s where added (perhaps a hidden SR friendly one), targeting a class might make it a little more future proof.